### ISSUE TITLE
Create a multi-node bridged network version of the GoEPC use case

### ISSUE THREAD

**Issue by taylor** [cncf/cnf-testbed#330]
# Multi-node bridged network GoEPC userplane use case using Multus

## This is a POC step in reaching the goal of a [multi-node segmented and routed user plane](https://github.com/cncf/cnf-testbed/milestone/56)

Tasks:

- [x] Create a single-node bridged network version #327 
- [x] Create documentation for K8s deployment 
- [x]  Update Helm charts to deploy EPC components to separate nodes. See: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
    - [ ] Add config option for enabling multi-host deployment.
                e.g ```helm install -e multihost=true```
- [x] Add a new node group to https://github.com/crosscloudci/infra-provisioning/blob/node_groups/terraform/packet.tf - So that we can support different worker node machine types in the same cluster.
- [x] Add a host interface to the Multus bridges. - So Pods will have connectivity over multiple hosts.
  - [x] Create automation code for adding the host interface to the Multus bridges.

---

### Testing: 

- [ ]  [Acceptance Criteria](https://github.com/cncf/cnf-testbed/issues/330#issuecomment-582503273)

**Comment by taylor** [comment]
### Acceptance Criteria:

Prereqs:
- Deploy K8s cluster with multus support following the [deployment instructions](https://github.com/cncf/cnf-testbed/tree/go-gtp/examples/use_case/gogtp-k8s/k8s_multus#prerequisites)
- Apply multus fix See https://github.com/cncf/cnf-testbed/tree/go-gtp/examples/use_case/gogtp-k8s/k8s_bridged_networks#prerequisites
- Clone the cnf-testbed
- Switch to the tag `k8s_bridged_network-v0.1` (`git checkout k8s_bridged_network-v0.1`)

**Comment by denverwilliams** [comment]
Tasks:

- [] Update Helm charts to deploy EPC components to separate nodes.
https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector

- [] Add new node group to https://github.com/crosscloudci/infra-provisioning/blob/node_groups/terraform/packet.tf - So that we can support different worker node types in the same cluster.

-[] Multus CNI bridge needs to use a host interface for connectivity to the other worker nodes.
     - Create automation code for adding the host Nic to the Multus bridge for each worker host.

**Comment by michaelspedersen** [comment]
The initial implementation for this has been added to the go-gtp branch (available in PR).

The Helm charts are in a separate directory, `examples/use_case/gogtp-k8s/k8s_multi_node`, so deployment is done using `helm install ./gogtp/`  instead of `helm install -e multihost=true`. This was a deliberate choice to avoid impacting existing code prior to having the new implementation added.

Also, the host environment must be configured prior to running the multi-node example. I have created an ansible playbook that works similar to how we have been configuring for use with vSwitch (VPP) previously. I have not yet updated the Makefile to simplify this step, but it should be easy to expand the existing vswitch case to also include the multinode config (e.g. using `||` (OR) and pointing towards the correct playbook). Similar to the Helm charts, I decided to push this initial version to make sure it is available in Github prior to integrating it with existing code.
